sirepo/package_data/template/srw/rsoptExport.sh.jinja

```diff
@@ -1,11 +1,27 @@
 #!/bin/bash
-echo "NOTE: remove files from {{ runDir }} first"
+if [ -d "{{ runDir }}" ] && [ -n "$(ls -A {{ runDir }})" ]; then
```

Comment:
It's better to do this if this is your intention:

```bash
for f in '{{ pyFileName }}' run_parallel_python.py; do
    rm -f "{{ runDir }}/$f"
done
```    

The output "Cleaning run directory {{ runDir }}..." is not useful. It's already happened so too late for the user. If you want to ask the user, then by all means prompt. Otherwise, they just lost data, and you don't have any way to recover.

The `-f` is important your code, too, because those files may or may not exist. You only know that the directory is not empty. That could be anything, the `rm` will output a message if the file doesn't exist, but not if `-f`.

The iteration is necessary only because there would be too many braces with:

`rm -f '{{ runDir }}/{{{ pyFileName }},run_parallel_python.py}`

which is the form I would like to use, and would use it if pyFileName was a shell variable, for example.

Reply:
dea7ee1